docs/design-docs/02-custom-reconciler-bindings.md

```diff
@@ -0,0 +1,247 @@
+# Custom RBAC assignments for `RootSync` and `RepoSync`
+
+* Author(s): @tomasaschan, @sdowell
+* Approver: @mortent, @karlkfi
+* Status: approved
+
+## Summary
+
+This proposal introduces a way to manage the lifecycle of RBAC bindings for both
+`RootSync` and `RepoSync` objects.
+
+When creating a reconciler deployment for a `RootSync`, Config Sync currently [creates a
```

Comment:
I think these 2 paragraphs might make more sense as a `Historical Behavior` section, followed by a `Proposed Behavior` section which briefly explains the new behavior.